Cython/Compiler/Code.py

```diff
@@ -1670,7 +1670,7 @@ def normalise_encoding_name(py_string):
         encodings.sort(key=len)  # stable sort to make sure '0' comes first, index 0
         assert not encodings or '0' not in encodings or encodings[0] == '0', encodings
         encodings_map = {encoding: i for i, encoding in enumerate(encodings)}
-        w.putln("static const char* %s[] = { %s };" % (
+        w.putln("static const char * const %s[] = { %s };" % (
```

Comment:
This change makes no difference (the binary size is the exact same number of bytes) but it's doesn't hurt to include it I think